src/visualization/types/Graph/view.tsx

```diff
@@ -73,6 +77,13 @@ const XYPlot: FC<Props> = ({
   const tooltipOrientationThreshold = properties.legendOrientationThreshold
   const staticLegend = useStaticLegend(properties)
   const dispatch = useDispatch()
+  const view = useSelector((state: AppState) =>
+    getByID<View<XYViewProperties>>(state, ResourceType.Views, cellID)
+  )
+  const {activeTimeMachineID} = useSelector(
+    (state: AppState) => state.timeMachines
+  )
+  const veoOpen = activeTimeMachineID === 'veo'
```

Comment:
maybe consider calling this something like `isVeoOpen` so it's not confused with a function. without seeing this line `veoOpen` might be construed as being a verb/action.